### PR TITLE
[Gekidou] RN patch

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Removes the patch added to `FlatList`, the patch was memoizing a few things that cause some of the items not to respond to touch gestures like press and long press and the performance gain was in fact degrading as the list was being scrolled causing it to not scroll smooth at times.

```release-note
NONE
```

### CODE REVIEW

**Review by larkox — APPROVED** [review]

File: patches/react-native+0.68.2.patch [hunk 1]
```diff
@@ -34,100 +34,6 @@ index a6826f3..a7b904b 100644
        child.props.onLayout(event);
      }
    };
-diff --git a/node_modules/react-native/Libraries/Lists/FlatList.js b/node_modules/react-native/Libraries/Lists/FlatList.js
```
Comment (avinashlng1080) [inline patches/react-native+0.68.2.patch:37]: I'm not following this file properly, can you please point to the fix in it?

**Review by avinashlng1080 — COMMENTED** [review]
Half done

File: patches/react-native+0.68.2.patch (hunk 1 above)
Comment (enahum) [reply patches/react-native+0.68.2.patch:37]: This is all being removed, it was added in a previous commit taken from a PR in the RN repo

**Comment by larkox** [comment]
Any chance this is solved in the 0.70 release of RN? If not, should we dropping an issue on their repo? (I know we are not exactly using 0.70, so we haven't properly tested it is not working, but still...)

**Comment by enahum** [comment]
> Any chance this is solved in the 0.70 release of RN? If not, should we dropping an issue on their repo? (I know we are not exactly using 0.70, so we haven't properly tested it is not working, but still...)

I've not looked into it really...

**Pull request merged**